client/state/login/reducer.js

```diff
@@ -101,8 +110,12 @@ export const requestSuccess = createReducer( null, {
 	[ LOGIN_REQUEST ]: () => null,
 	[ LOGIN_REQUEST_SUCCESS ]: () => true,
 	[ LOGIN_REQUEST_FAILURE ]: () => false,
+	[ SOCIAL_CREATE_ACCOUNT_REQUEST ]: () => null,
 	[ SOCIAL_CREATE_ACCOUNT_REQUEST_SUCCESS ]: () => true,
+	[ SOCIAL_CONNECT_ACCOUNT_REQUEST ]: () => null,
 	[ SOCIAL_CONNECT_ACCOUNT_REQUEST_SUCCESS ]: () => true,
+	[ SOCIAL_DISCONNECT_ACCOUNT_REQUEST ]: () => null,
+	[ SOCIAL_DISCONNECT_ACCOUNT_REQUEST_SUCCESS ]: () => true,
 } );
 
```

Comment:
Should we also update the `socialAccountLink` reducer to take `SOCIAL_DISCONNECT_ACCOUNT_REQUEST_SUCCESS` into account?

Reply:
Well `socialAccountLink` is about the socialAccount linking *view* we have on the login page, so we keep the social info in store until the user logs in. Here the user is already logged in so we don't need to update that store imo.